src/controllers/userController.js

```diff
@@ -0,0 +1,35 @@
+import { Subscribes } from "../database";
+
+class userController {
+
+    static async subscribe(req,res){
+        const {
```

Comment:
Please, you can add validation of username and email, when all are empty or one of them
Please send the appropriate message for the missing attribute.